tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -124,53 +124,113 @@ def get_virtctl():
+                log.info(f"Storage class {sc_name} already exists")
+                continue
+            try:
+                sc_obj = storageclass_factory(
+                    sc_name=sc_name,
+                    pool_name=pool_name,
+                    mapOptions="krbd:rxbounce",
+                )
+                if sc_obj is None or sc_obj.name != sc_name:
+                    log.error(
+                        f"Failed to create SC '{sc_name}' or name mismatch: "
+                        f"Created '{sc_obj.name if sc_obj else 'None'}'"
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)